testsuite/elytron/src/test/java/org/wildfly/test/security/common/elytron/MappedRoleMapperTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.test.security.common.elytron;
+
+import static org.hamcrest.core.StringContains.containsString;
+
+import org.hamcrest.MatcherAssert;
+import org.jboss.as.test.integration.management.util.CLIWrapper;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.WildFlyRunner;
+
+@RunWith(WildFlyRunner.class)
+public class MappedRoleMapperTest {
+
+    CLIWrapper cli;
+
+    @Before
+    public void setup() throws Exception {
+        cli = new CLIWrapper(true);
+    }
+
+    @After
+    public void cleanup() throws Exception {
+        cli.sendLine("reload", true);
```

Comment:
We also need to make sure that all of the resources that were added in this test get removed.